include/tester/ssp/SSPCall.hpp

```diff
@@ -68,4 +63,16 @@ class SSPCall : public pj::Call
         /* Create new Call for call replace */
         prm.newCall = new SSPCall(*_myAcc);
     }
+    
+    void callTo(std::string & destination_uri)
+    {
+        pj::CallOpParam prm(true);
+        prm.opt.audioCount = 1;
+        prm.opt.videoCount = 0;
+        makeCall(destination_uri, prm);
+    }
+
+private:
+    SSPAccount *_myAcc;
```

Comment:
There's no usage of it anywhere. Remove this member.

Reply:
I 2 function how use this member